### ISSUE TITLE
Crash on world creation

### ISSUE THREAD

**Issue by DDZiroWan** [28Pollux28/ImageWorldGenerator#1]
```
[15:30:31] [Server thread/WARN]: Unable to find spawn biome
[15:30:31] [Server thread/ERROR]: Encountered an unexpected exception
net.minecraft.class_148: Exception initializing level
	at net.minecraft.server.MinecraftServer.method_3786(MinecraftServer.java:390) ~[intermediary-minecraft-1.16.1-client.jar:?]
	at net.minecraft.server.MinecraftServer.method_3735(MinecraftServer.java:334) ~[intermediary-minecraft-1.16.1-client.jar:?]
	at net.minecraft.class_1132.method_3823(class_1132.java:89) ~[intermediary-minecraft-1.16.1-client.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:647) ~[intermediary-minecraft-1.16.1-client.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:256) ~[intermediary-minecraft-1.16.1-client.jar:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NullPointerException
	at net.minecraft.class_5322.method_29194(class_5322.java:21) ~[intermediary-minecraft-1.16.1-client.jar:?]
	at net.minecraft.class_5322.method_29196(class_5322.java:62) ~[intermediary-minecraft-1.16.1-client.jar:?]
	at net.minecraft.server.MinecraftServer.method_27901(MinecraftServer.java:457) ~[intermediary-minecraft-1.16.1-client.jar:?]
	at net.minecraft.server.MinecraftServer.method_3786(MinecraftServer.java:378) ~[intermediary-minecraft-1.16.1-client.jar:?]
	... 5 more
[15:30:31] [Server thread/ERROR]: This crash report has been saved to: /home/diegoilcatanico/.local/share/multimc/instances/1.16/.minecraft/crash-reports/crash-2020-08-27_15.30.31-server.txt
[15:30:31] [Server thread/ERROR]: Exception stopping the server
java.lang.NullPointerException: null
	at com.google.common.base.Preconditions.checkNotNull(Preconditions.java:770) ~[guava-21.0.jar:?]
	at com.sk89q.worldedit.extension.platform.PlatformManager.unregister(PlatformManager.java:125) ~[worldedit-fabric-mc1.16.1-7.2.0-beta-02.jar:?]
	at com.sk89q.worldedit.fabric.FabricWorldEdit.onStopServer(FabricWorldEdit.java:206) ~[worldedit-fabric-mc1.16.1-7.2.0-beta-02.jar:?]
	at net.fabricmc.fabric.api.event.lifecycle.v1.ServerLifecycleEvents.lambda$null$4(ServerLifecycleEvents.java:62) ~[257e270f-73c7-4857-b2e9-c003b2177591.jar:?]
	at net.minecraft.server.MinecraftServer.handler$zzo000$beforeShutdownServer(MinecraftServer.java:1734) ~[intermediary-minecraft-1.16.1-client.jar:?]
	at net.minecraft.server.MinecraftServer.method_3782(MinecraftServer.java) ~[intermediary-minecraft-1.16.1-client.jar:?]
	at net.minecraft.class_1132.method_3782(class_1132.java:244) ~[intermediary-minecraft-1.16.1-client.jar:?]
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:709) ~[intermediary-minecraft-1.16.1-client.jar:?]
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:256) ~[intermediary-minecraft-1.16.1-client.jar:?]
	at java.lang.Thread.run(Thread.java:834) [?:?]
---- Minecraft Crash Report ----
// Don't be sad, have a hug! <3

Time: 8/27/20, 3:30 PM
Description: Exception initializing level

java.lang.NullPointerException: Exception initializing level
	at net.minecraft.class_5322.method_29194(class_5322.java:21)
	at net.minecraft.class_5322.method_29196(class_5322.java:62)
	at net.minecraft.server.MinecraftServer.method_27901(MinecraftServer.java:457)
	at net.minecraft.server.MinecraftServer.method_3786(MinecraftServer.java:378)
	at net.minecraft.server.MinecraftServer.method_3735(MinecraftServer.java:334)
	at net.minecraft.class_1132.method_3823(class_1132.java:89)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:647)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:256)
	at java.base/java.lang.Thread.run(Thread.java:834)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_5322.method_29194(class_5322.java:21)
	at net.minecraft.class_5322.method_29196(class_5322.java:62)
	at net.minecraft.server.MinecraftServer.method_27901(MinecraftServer.java:457)

-- Affected level --
Details:
	All players: 0 total; []
	Chunk stats: ServerChunkCache: 0
	Level dimension: minecraft:overworld
	Level spawn location: World: (8,64,8), Chunk: (at 8,4,8 in 0,0; contains blocks 0,0,0 to 15,255,15), Region: (0,0; contains chunks 0,0 to 31,31, blocks 0,0,0 to 511,255,511)
	Level time: 0 game time, 0 day time
	Level name: New World
	Level game mode: Game mode: creative (ID 1). Hardcore: false. Cheats: true
	Level weather: Rain time: 0 (now: false), thunder time: 0 (now: false)
	Known server brands: fabric
	Level was modded: true
	Level storage version: 0x04ABD - Anvil
Stacktrace:
	at net.minecraft.server.MinecraftServer.method_3786(MinecraftServer.java:378)
	at net.minecraft.server.MinecraftServer.method_3735(MinecraftServer.java:334)
	at net.minecraft.class_1132.method_3823(class_1132.java:89)
	at net.minecraft.server.MinecraftServer.method_29741(MinecraftServer.java:647)
	at net.minecraft.server.MinecraftServer.method_29739(MinecraftServer.java:256)
	at java.base/java.lang.Thread.run(Thread.java:834)

-- System Details --
Details:
	Minecraft Version: 1.16.1
	Minecraft Version ID: 1.16.1
	Operating System: Linux (amd64) version 5.4.0-42-generic
	Java Version: 11.0.8, Ubuntu
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Ubuntu
	Memory: 408535712 bytes (389 MB) / 1305477120 bytes (1245 MB) up to 2147483648 bytes (2048 MB)
	CPUs: 12
	JVM Flags: 2 total; -Xms512m -Xmx2048m
	Fabric Mods: 
		autoconfig1u: Auto Config v1 Updated 3.2.0-unstable
		cloth-basic-math: Cloth Basic Math 0.5.1
		cloth-config2: Cloth Config v4 4.6.0
		enchantmentnumeralfix: Enchantment Numeral Fix 1.0.0
		fabric: Fabric API 0.15.1+build.380-1.16.1
		fabric-api-base: Fabric API Base 0.1.3+12a8474c02
		fabric-biomes-v1: Fabric Biomes (v1) 0.2.7+059ea86602
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.4+c6a8ea8902
		fabric-command-api-v1: Fabric Command API (v1) 1.0.8+5ce5339802
		fabric-commands-v0: Fabric Commands (v0) 0.2.0+52d3083602
		fabric-containers-v0: Fabric Containers (v0) 0.1.8+045df74f02
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.1.9+059ea86602
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.2+b7f9825d02
		fabric-dimensions-v1: fabric-dimensions-v1 1.0.0+a71b305302
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.3.3+7066030f02
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.0+16acbe5b02
		fabric-item-api-v1: Fabric Item API (v1) 1.0.0+16acbe5b02
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.0+438f963602
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.1+f404f3be02
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.0+3fa9f7c502
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.1.0+013e49d202
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.0+386eb69e02
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.2+b764ce9902
		fabric-models-v0: Fabric Models (v0) 0.1.0+dfdb52d602
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.5+b50ffc7b02
		fabric-networking-v0: Fabric Networking (v0) 0.1.10+e00ecb5f02
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.5.6+a2d21ddd02
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.6.1+a2d21ddd02
		fabric-particles-v1: fabric-particles-v1 0.2.1+0a6f2a7002
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.3.8+7dba2d6c02
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.2.13+eae12eb802
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.3.3+0a55f37902
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 2.0.1+5a0f9a6002
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.3+b7f9825d02
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.8+b7084faa02
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.0+5341049002
		fabric-rendering-v1: Fabric Rendering (v1) 1.1.2+346247d702
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.2.6+f41e209802
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.0+97324d1102
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 0.2.7+a4c57d8e02
		fabric-textures-v0: Fabric Textures (v0) 1.0.4+eae12eb802
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.1.4+5794386e02
		fabricloader: Fabric Loader 0.9.0+build.204
		fiber: fiber 0.23.0-1
		flighthud: Flight HUD 1.16.1-fabric-4
		imggen: Image World Generator 1.0.0
		minecraft: Minecraft 1.16.1
		mm: Manningham Mills 2.0
		modmenu: Mod Menu 1.14.5+build.30
		okzoomer: Ok Zoomer 4.0.0-alpha.4.1.16.1
		optifabric: Optifabric 1.1.0-beta2
		org_slf4j_slf4j-api: slf4j-api 1.7.30
		org_slf4j_slf4j-simple: slf4j-simple 1.7.30
		org_zeroturnaround_zt-zip: zt-zip 1.14
		worldedit: WorldEdit 7.2.0-beta-02+d10d7d6
	Player Count: 0 / 8; []
	Data Packs: vanilla, fabric/fabric-tool-attribute-api-v1
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fabric'
	OptiFine Version: OptiFine_1.16.1_HD_U_G2
	OptiFine Build: 20200802-005745
	Render Distance Chunks: 10
	Mipmaps: 4
	Anisotropic Filtering: 1
	Antialiasing: 0
	Multitexture: false
	Shaders: BSL_v7.1.05p3.zip
	OpenGlVersion: 4.6.0 NVIDIA 440.100
	OpenGlRenderer: GeForce GTX 1660/PCIe/SSE2
	OpenGlVendor: NVIDIA Corporation
	CpuCount: 12
```

**Comment by DDZiroWan** [comment]
![512x512 world](https://user-images.githubusercontent.com/21110014/91449203-1dfd9180-e87b-11ea-8404-453b1c9693fc.png)

**Comment by 28Pollux28** [comment]
are you on client or server ?

**Comment by 28Pollux28** [comment]
what config options did you use ?

**Comment by 28Pollux28** [comment]
Ok will be fixed normally in 0.1.1.

**Comment by 28Pollux28** [comment]
here you go :) https://www.curseforge.com/minecraft/mc-mods/image-world-generator/files/3043484
